src/Daemon/BaseDaemon.cpp

```diff
@@ -543,6 +544,10 @@ class SignalListener : public Poco::Runnable
 
         fatal_error_printed.test_and_set();
     }
+    catch (...)
+    {
+        tryLogCurrentException(__PRETTY_FUNCTION__);
```

Comment:
Consider logging this with `Fatal` level as well

Reply:
OK, make sense. Done.